build.sbt

```diff
@@ -12,7 +12,7 @@ lazy val headerMsg =  """/** This software is released under the University of I
                         |""".stripMargin
 
 lazy val root = (project in file(".")).
-  aggregate(saulCore, saulExamples, saulWebapp).
```

Comment:
Two of the three tests in the web-app are failing. They are very basic tests on the web application.